src/C0_data_processing.py

```diff
@@ -203,13 +203,22 @@ def energyConversion(dict_values, group):
     """
     #
     for asset in dict_values[group]:
+        print("asset", asset)
         define_missing_cost_data(dict_values, dict_values[group][asset])
         evaluate_lifetime_costs(
             dict_values["simulation_settings"],
             dict_values["economic_data"],
             dict_values[group][asset],
         )
 
+        if asset == "electrolyser":
```

Comment:
```suggestion
        if asset == "electrolyser" or asset == "Electrolyzer":
```
This info - that the function is turned on a bit messily over the exact names "electrolyzer" or "Electrolyzer" should be mentioned in issue #50